statsd/src/main/java/com/avast/metrics/statsd/StatsDMeter.java

```diff
@@ -0,0 +1,41 @@
+package com.avast.metrics.statsd;
+
+import com.avast.metrics.api.Meter;
+import com.timgroup.statsd.StatsDClient;
+
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class StatsDMeter implements Meter {
```

Comment:
what is the difference between meter and counter? That you cannot decrement meter?